03/GeoffRiley/tags.py

```diff
@@ -0,0 +1,41 @@
+import re
+from collections import Counter
+from difflib import SequenceMatcher
+from itertools import permutations
+
+TOP_NUMBER = 10
+RSS_FEED = 'rss.xml'
+SIMILAR = 0.87
+
+TAG_REGEX = re.compile(r'<category>([^<]+?)</category>')
+
+
+def get_tags() -> list:
+    """Find all tags in RSS_FEED.
+    Replace dash with whitespace."""
+    with open(RSS_FEED) as f:
+        return [t.replace('-', ' ').lower() for t in TAG_REGEX.findall(f.read())]
+
+
+def get_top_tags(tags):
```

Comment:
how many are the top tags? Maybe improve function name to show that the function returns a variable amount, use your constant as default: `def get_n_most_common_tags(tags, n=TOP_NUMBER):`

Reply:
`get_top_tags(tags)` was provided as part of the boilerplate for the solution, as with the other routine names and constants.  Since I was working from the 'no-help' version of the template I assumed that these were to be taken as writ.

Reply:
Oh then I am very sorry and appologize for critizing it, or better for critizing you instead of the real culprit haha. That applies to all other comments as well. So there is not much then I can comment anyway ;)

Reply:
No worries; I'm thick skinned enough to take much harsher peer review comments than this! :rofl: